components/layout/Footer.tsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react'
 import TwitterIconSvg from '../../public/svg/twitter-icon.svg'
+import XIconSvg from '../../public/svg/x-icon.svg'
 import GithubIconSvg from '../../public/svg/github-icon.svg'
+import LinkedInIconSvg from '../../public/svg/linkedin-icon.svg'
+import YoutubeIconSvg from '../../public/svg/youtube-icon.svg'
```

Comment:
This is fine... but I think if we change these again we should just use a Icon library eg. [Tabler](https://tablericons.com/)

Reply:
Noted